File: templates/Uwp/_composition/_shared/Page.Ink.AddBasicInkServices/EventHandlers/Ink/MoveStrokesEventArgs.cs
using System.Collections.Generic;
using Windows.Foundation;
using Windows.UI.Input.Inking;

namespace Param_ItemNamespace.EventHandlers.Ink
{
    public class MoveStrokesEventArgs
    {
        public Point StartPosition { get; set; }

        public Point EndPosition { get; set; }

        public IEnumerable<InkStroke> Strokes { get; set; }

        public MoveStrokesEventArgs(IEnumerable<InkStroke> strokes, Point startPosition, Point endPosition)
        {
            Strokes = strokes;
            StartPosition = startPosition;
            EndPosition = endPosition;
        }
    }
}

File: templates/Uwp/_composition/_shared/Page.Ink.AddBasicInkServicesVB/EventHandlers/Ink/MoveStrokesEventArgs.vb
Imports Windows.UI.Input.Inking

Namespace EventHandlers.Ink
    Public Class MoveStrokesEventArgs

        Public Property StartPosition As Point

        Public Property EndPosition As Point

        Public Property Strokes As IEnumerable(Of InkStroke)

        Public Sub New(strokes As IEnumerable(Of InkStroke), startPosition As Point, endPosition As Point)
            Me.Strokes = strokes
            Me.StartPosition = startPosition
            Me.EndPosition = endPosition
        End Sub
    End Class
End Namespace
